react/index.js

```diff
@@ -1,26 +1,19 @@
 import PropTypes from 'prop-types'
-import { path } from 'ramda'
 import React, { Component } from 'react'
-import { Link, withRuntimeContext } from 'vtex.render-runtime'
-import classNames from 'classnames'
+import { withRuntimeContext } from 'vtex.render-runtime'
 import {
   ProductName,
   ProductPrice,
 } from 'vtex.store-components'
 
-import AttachmentList from './components/AttachmentList'
-import ImageLoader from './components/ImageLoader'
-import ProductImage from './components/ProductImage'
-import ProductSummaryBuyButton from './components/ProductSummaryBuyButton'
-import ProductQuantityStepper from './components/ProductQuantityStepper'
-import ProductSummaryPrice from './components/ProductSummaryPrice'
-import ProductSummaryName from './components/ProductSummaryName'
+import ProductSummaryNormal from './components/ProductSummaryNormal'
```

Comment:
we could use `React.lazy` for code splitting in  the future for these imports 🤔

Reply:
I think we can use now, Render 8 has the react version with this feature.

Reply:
I got the following errors trying to use `React.lazy`
```
"name":"Invariant Violation","framesToPop":1,"message":"ReactDOMServer does not yet support lazy-loaded components.","stack":"Invariant Violation: ReactDOMServer does not yet support lazy-loaded components.
```